src/views/Home/components/Milestones/index.scss

```
@@ -178,16 +185,20 @@
     align-items: center;
     margin: 2% 0 2% 0;
 
-    &__image {
+    &__image-container {
       width: 180px;
+    }
 
-      > img {
-        margin-right: 10px;
-        width: 75%;
-        float: right;
+    &__milestone-img {
```

I think you meant to do this?

```suggestion
    &-img {
```

Or

```suggestion
    &__img {
```
and change the class name to `milestones-component__milestone__img` below.